swift-sdk/Internal/Dwifft/Dwifft.swift

```diff
@@ -308,18 +308,17 @@ fileprivate struct MemoizedSequenceComparison<T: Equatable> {
 
 
 // MARK: - Deprecated
```

Comment:
We should either remove it or keep it. Commented out code does not look very nice.